src/inmanta/protocol/auth/providers.py

```diff
@@ -148,6 +148,9 @@ async def _do_authorize_request(self, call_arguments: "rest.CallArguments") -> N
             # On a login call the authorization is done using the provided username and password.
             # No need to authorize the token.
             return
+        if call_arguments.method_properties.function == methods_v2.health:
+            # For ease of use, the health endpoint should be accessible without authentication.
+            return
```

Comment:
What's the rationale for configuring this here vs in the (default) policy file?

Reply:
You make a good argument. I was only vaguely aware of the purpose of this endpoint. In that case I think it's more a property of the system than a policy, so I think it belongs here after all.